db/system_keyspace.cc

```diff
@@ -2089,7 +2102,7 @@ std::vector<schema_ptr> system_keyspace::all_tables(const db::config& cfg) {
                     v3::cdc_local(),
     });
 
-    r.insert(r.end(), {raft(), raft_snapshots(), raft_snapshot_config(), group0_history(), discovery()});
+    r.insert(r.end(), {raft(), raft_snapshots(), raft_snapshot_config(), group0_history(), discovery(), service_levels_v2()});
```

Comment:
We should only add experimental features where strictly required. This should be automatic and unconditional. If we rely on infrastructure developed for raft topology, then it can depend on that.

We will need a cluster feature so the upgrade only starts after all nodes are ready.

Reply:
I've dropped the experimental feature, now raft service levels are used when `consistent-topology-changes` is on

Reply:
In that case, shouldn't it be under `if (cfg.check_experimental(db::experimental_features_t::feature::CONSISTENT_TOPOLOGY_CHANGES)) { ... }`?

Reply:
moved